Directory.Packages.props

```diff
@@ -127,6 +127,7 @@
     <!-- unit test dependencies -->
     <PackageVersion Include="bUnit" Version="1.26.64" />
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="$(MicrosoftExtensionsDiagnosticsTestingPackageVersion)" />
+    <PackageVersion Include="Testcontainers.RabbitMq" Version="3.7.0" />
```

Comment:
It's the first time we use these packages, should we do it more often?

Reply:
Yes, I'm in the middle of converting the other RabbitMQ tests to use this package.